src/mixins/base-mixin.js

```diff
@@ -484,6 +484,7 @@ export default function baseMixin(_chart) {
       }
       _root = d3.select(_anchor)
       _root.classed(constants.CHART_CLASS, true)
+      deregisterChart(_chart, chartGroup)
```

Comment:
This is in the base-mixin - does this bug actually apply to every chart?

Reply:
@jrajav, as I noticed, this bug also exists in Pie chart. I checked couple other charts which we can switch dimensions such as Bar chart but the bug is not there. I wonder if it is protected somewhere. Anyway, the above function only deregister the current chart, then we register it right after, so it should be safe. I will ask QA to test other thoroughly.